packages/template-retail-react-app/package.json

```diff
@@ -18,15 +18,19 @@
         "@lhci/cli": "^0.9.0",
         "@loadable/component": "^5.15.3",
         "@peculiar/webcrypto": "^1.4.2",
+        "@salesforce/commerce-sdk-react": "file:../../../pwa-kit/packages/commerce-sdk-react/src",
```

Comment:
Until the commerce-sdk-react package is released, I guess we can leave this as is for now.